conandata.yml

```diff
@@ -1,17 +1,17 @@
-version: "5.7.0-alpha.1"
+version: "5.7.0-beta.1"
 requirements:
-  - "uranium/(latest)@ultimaker/testing"
-  - "curaengine/(latest)@ultimaker/testing"
-  - "cura_binary_data/(latest)@ultimaker/testing"
-  - "fdm_materials/(latest)@ultimaker/testing"
-  - "curaengine_plugin_gradual_flow/0.1.0-beta.3"
-  - "dulcificum/latest@ultimaker/testing"
+  - "uranium/(latest)@ultimaker/stable"
+  - "curaengine/(latest)@ultimaker/stable"
+  - "cura_binary_data/(latest)@ultimaker/stable"
+  - "fdm_materials/(latest)@ultimaker/stable"
+  - "curaengine_plugin_gradual_flow/0.1.0-beta.3@ultimaker/stable"
+  - "dulcificum/latest@ultimaker/stable"
```

Comment:
we should only pin to stable for repos that have release branches, i believe `dulcificum` doesn't have a release branch?

Reply:
No indeed, but I thought we would manually build a package on the stable channel ?

Reply:
Creating a Conan package in the cura-workflows is only used for a specific package. Such as a public release. We are now simply redirecting traffic from the `testing` branches to the `stable` branches.